features/works/work_edit.feature

```diff
@@ -75,9 +75,9 @@ Feature: Edit Works
     # Test changing pseuds on a work
     When I go to testuser's works page
       And I follow "Edit"
-      And I select "testy" from "work_author_attributes_ids"
-      And I unselect "testuser" from "work_author_attributes_ids"
-      And I press "Post Without Preview"
+      And I select "testy" from "work_author_attributes_ids_"
```

Comment:
I think the test failure is due to this and line 79 -- they should be `work_author_attributes_ids`.